### PR TITLE
Add icon for HiveMQ

### LINKED ISSUE #10336: Request: HiveMQ and products
### Brand Name

- [x] HiveMQ - #10393
- [ ] HiveMQ MQTT Client
- [ ] HiveMQ Edge
- [ ] HiveMQ Cloud

### Website

https://www.hivemq.com/

### Popularity Metric

# Similarweb Popularity Metric
The Similarweb rank of HiveMQ is #321,877 as of submitting this issue: https://www.similarweb.com/website/hivemq.com/

### Official Resources for Icon and Color

# HiveMQ Logos

The following are the primary images of the HiveMQ logos I can find on the internet.

image
image
image

# Media Kit and Official Website
**Old Link of their media kit**: https://www.hivemq.com/logos-and-media-kit
Official Website: https://www.hivemq.com/
GitHub: https://github.com/hivemq

### Additional Comments

# HiveMQ
> HiveMQ is the proven enterprise MQTT platform because it’s reliable under real-world stress and built for flexibility, security, and scale.

# Things to consider... 
There are other variants of their logo relative to their family of products connected to MQTT.

They also have HiveMQ MQTT Client which has a "beehive" logo, but with the text. Perhaps we could have a logo with just the beehive.
image

the https://www.hivemq.com/logos-and-media-kit links works before, but now it currently redirects to the official HiveMQ website.

Another product that they have is HiveMQ Edge: Open-source IIoT gateway & protocol converter to MQTT, which is a variant of the original logo, but uses a heptagon instead of a circular shaped logo.
image

Another logo is for HiveMQ Cloud
image

### PR DESCRIPTION
**Pull request by jeoooo**
# Add icon for HiveMQ
image

**Issue:** closes #10336 

**Similarweb rank: 321,877** 

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

- HiveMQ is one of the world's leading companies within the IoT sector for connecting machines, devices, and applications.
- Logo can be found in their branding guidelines: https://www.hivemq.com/company/hivemq-brand-resources/
- Guidelines can also be found in their branding guidelines: https://www.hivemq.com/company/hivemq-brand-resources/
- Other than the HiveMQ logo we can consider adding other HiveMQ-related product logos for Simple Icons in separate GitHub issues (and separate PRs). It was discussed in #10336. For now, this will close issue #10336

### CODE REVIEW

**Comment by adamrusted** [comment]
There is a monochrome variant  provided at `HiveMQ_Logos.zip\HiveMQ Logos\HiveMQ Logo\sw\01-hivemq-sw.svg`. Could you re-tool this PR to use that logo instead please @jeoooo?

**Comment by jeoooo** [comment]
@adamrusted  

Is that the logo that has the text "HiveMQ" attached to it? ~would having text to logos be okay?~

if so, I think just the logo without the text would be enough.

**Comment by adamrusted** [comment]
@jeoooo yeah, it might have text? I don't have the file open right now. Regardless, we can bin off the text if the icon is used without elsewhere 👌

**Comment by jeoooo** [comment]
@adamrusted  

well according to their website: https://www.hivemq.com/company/hivemq-brand-resources/

> # Primary Logo
> The HiveMQ logo is the face of our brand. It’s a symbol that represents our company and all it has to offer. To support our focus on clarity, we have shifted away from the use of unique logos for our services and product offers.

image

> # Logo Mark
> Our iconic bee logo can be used in isolation, but must be in context where the HiveMQ Logo has already been shown or it is clear the audience understands the reference of our full logo.

image

# TLDR: 
**bee logo is fine for now**, 
if any problems (such as HiveMQ taking notice of this repo, and suggests replacing it with the text variant) arise, then we can change it to bee + text variant.

**Comment by jeoooo** [comment]
sounds great!

will contribute more to this repo. this sounds nice! 

~ps. can I add fillipino brands?~

**Comment by adamrusted** [comment]
> ps. can I add fillipino brands?

For sure, yeah - providing they meet [our metrics for inclusion](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon).

**Review by adamrusted — APPROVED** [review]
